app/src/main/java/com/orange/ods/app/ui/AppBarTabsState.kt

```diff
@@ -69,18 +72,20 @@ class AppBarTabsState(
 @Composable
 fun rememberAppBarTabsState(
     tabs: SnapshotStateList<NavigationItem> = rememberSaveableMutableStateListOf(),
-    tabIconType: MutableState<MainTabsCustomizationState.TabIconType> = rememberSaveable { mutableStateOf(AppBarTabsState.DefaultConfiguration.tabIconType) },
+    tabsIconPosition: MutableState<MainTabsCustomizationState.TabsIconPosition> = rememberSaveable { mutableStateOf(AppBarTabsState.DefaultConfiguration.tabsIconPosition) },
+    tabIconEnabled: MutableState<Boolean> = rememberSaveable { mutableStateOf(AppBarTabsState.DefaultConfiguration.tabIconEnabled) },
     tabTextEnabled: MutableState<Boolean> = rememberSaveable { mutableStateOf(AppBarTabsState.DefaultConfiguration.tabTextEnabled) },
     scrollableTabs: MutableState<Boolean> = rememberSaveable { mutableStateOf(AppBarTabsState.DefaultConfiguration.scrollableTabs) }
-) = remember(tabs, tabIconType, tabTextEnabled, scrollableTabs) {
-    AppBarTabsState(tabs, tabIconType, tabTextEnabled, scrollableTabs)
+) = remember(tabs, tabsIconPosition, tabIconEnabled, tabTextEnabled, scrollableTabs) {
+    AppBarTabsState(tabs, tabsIconPosition, tabTextEnabled, tabIconEnabled, scrollableTabs)
```

Comment:
`tabIconEnabled` and `tabTextEnabled ` are inverted:

```suggestion
    AppBarTabsState(tabs, tabsIconPosition, tabIconEnabled, tabTextEnabled, scrollableTabs)
```